src/snapred/backend/service/CrystallographicInfoService.py

```diff
@@ -0,0 +1,29 @@
+from typing import Any, Dict
+
+from snapred.backend.data.DataFactoryService import DataFactoryService
+from snapred.backend.recipe.CrystallographicInfoRecipe import CrystallographicInfoRecipe
+from snapred.meta.Singleton import Singleton
+
+
+@Singleton
+class IngestionService:
```

Comment:
Could you refactor this to be in line with the new way we do services?
i.e. rename the orchestrate method, decorate it with FromString, and register the former orchestrate method in init to "", and have IngestionService inherit from Service